pom.xml

```diff
@@ -105,6 +105,7 @@
             <groupId>com.amazonaws</groupId>
             <artifactId>aws-java-sdk-s3</artifactId>
             <version>${aws.version}</version>
+            <optional>true</optional>
```

Comment:
Let's move this down to the junit dep and add a comment saying "JUnit Rule dependencies" or something like that.

Reply:
Done